samples/dashboards/demo/cloud-monitoring/demo.js

```diff
@@ -0,0 +1,724 @@
+                            '</div>',
+                            useHTML: true
+                        }
+                    }
+                },
+                series: [{
+                    name: 'Memory usage',
+                    innerRadius: '80%',
+                    data: [{
+                        colorIndex: '100'
+                    }],
+                    radius: '100%'
+                }],
+                lang: {
+                    accessibility: {
+                        chartContainerLabel: 'Memory usage. Highcharts interactive chart.'
+                    }
+                },
+                tooltip: {
+                    valueSuffix: ' MB'
+                }
+            }
+        },
+        {
+            cell: 'health',
+            type: 'HTML',
+            class: 'health-indicator',
+            elements: [{
+                tagName: 'div',
+                class: 'health-wrapper highcharts-' + instance.HealthIndicator + '-icon',
+                attributes: {
+                    'aria-label': 'Health: ' + instance.HealthIndicator,
+                    role: 'img'
+                }
+            }, {
+                tagName: 'div',
+                class: 'health-title',
+                textContent: 'Health'
+            }]
+        },
+        {
+            cell: 'disk',
+            type: 'KPI',
+            connector: {
+                id: 'instanceDetails'
+            },
+            columnName: 'DiskUsedGB',
+            chartOptions: {
+                ...KPIOptions,
+                plotOptions: {
+                    series: {
+                        dataLabels: {
+                            format: '<div style="text-align:center; margin-top: -20px">' +
+                            '<div style="font-size:1.2em;">{y} GB</div>' +
+                            '<div style="font-size:14px; opacity:0.4; text-align: center;">Disk space</div>' +
+                            '</div>',
+                            useHTML: true
+                        }
+                    }
+                },
+                series: [{
+                    name: 'Disk usage',
+                    innerRadius: '80%',
+                    data: [{
+                        colorIndex: '100'
+                    }],
+                    radius: '100%'
+                }],
+                tooltip: {
+                    valueSuffix: ' Gb'
+                },
+                lang: {
+                    accessibility: {
+                        chartContainerLabel: 'Disk usage. Highcharts interactive chart.'
+                    }
+                }
+            }
+        },
+        {
+            cell: 'network-opt',
+            type: 'Highcharts',
+            title: 'Network (bytes)',
+            connector: {
+                id: 'charts'
+            },
+            columnAssignment: {
+                timestamp: 'x',
+                networkIn: 'y',
+                networkOut: 'y'
```

Comment:
In both the _Network (bytes)_ chart and the _Disk operations_ chart, the column names are used for `series.name`. Is it possible to add more human readable series names (like "Network In", "Read operations") instead of the camelCase column headers? 

This may be a bigger issue though - according to the docs, `columnAssignment` matches series by their `name`. It would be preferable if they were matched by `series.id` instead, in consistence with other Highcharts matching (like `series.xAxis`, `series.linkedTo` etc). In general, the `name` property is intended for human readers, while the `id` is intended for computing. If this is not currently possible, it's not a blocker for this demo, but we should add an enhancement request for it.

Reply:
You are right, it should be matched by ID.

Meanwhile I used a simple snippet in the labelFormatter.